integration-tests/rpc_integration_test.go

```diff
@@ -0,0 +1,465 @@
+		PeerId   string `json:"remotePeerID_string"`
+		Protocol string `json:"protocol_string"`
+	}
+
+	err = json.Unmarshal([]byte(log), &foundPeerLog)
+	require.NoError(t, err)
+
+	parsedFoundPeerID2, err := peer.IDB58Decode(foundPeerLog.PeerId)
+	require.NoError(t, err)
+	assert.Equal(t, parsedFoundPeerID2, parsedPeerID2)
+	assert.Equal(t, foundPeerLog.Protocol, protocolString)
+
+	log, err = waitForLogSubstring(ctx, logMessages2, "found peer who speaks our protocol")
+	require.NoError(t, err, "didn't find peer")
+
+	err = json.Unmarshal([]byte(log), &foundPeerLog)
+	require.NoError(t, err)
+
+	parsedFoundPeerID1, err := peer.IDB58Decode(foundPeerLog.PeerId)
+	require.NoError(t, err)
+	assert.Equal(t, parsedFoundPeerID1, parsedPeerID1)
+	assert.Equal(t, foundPeerLog.Protocol, protocolString)
+
+	log, err = waitForLogSubstring(ctx, logMessages1, "found peer who speaks our protocol")
+	require.NoError(t, err, "didn't find peer")
+
+	err = json.Unmarshal([]byte(log), &foundPeerLog)
+	require.NoError(t, err)
+
+	parsedFoundPeerID2, err = peer.IDB58Decode(foundPeerLog.PeerId)
+	require.NoError(t, err)
+	assert.Equal(t, parsedFoundPeerID2, parsedPeerID2)
+	assert.Equal(t, foundPeerLog.Protocol, protocolString)
+
+	log, err = waitForLogSubstring(ctx, logMessages2, "found peer who speaks our protocol")
+	require.NoError(t, err, "didn't find peer")
+
+	err = json.Unmarshal([]byte(log), &foundPeerLog)
+	require.NoError(t, err)
+
+	parsedFoundPeerID1, err = peer.IDB58Decode(foundPeerLog.PeerId)
+	require.NoError(t, err)
+	assert.Equal(t, parsedFoundPeerID1, parsedPeerID1)
+	assert.Equal(t, foundPeerLog.Protocol, protocolString)
+
+	// Cancel the context and wait for all outstanding goroutines to finish.
+	cancel()
+	wg.Wait()
+}
+
+func TestGetStats(t *testing.T) {
+	teardownSubTest := setupSubTest(t)
+	defer teardownSubTest(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	// Remove the old database and p2p files.
+	removeOldFiles(t, ctx)
+
+	buildStandaloneForTests(t, ctx)
+
+	logMessages := make(chan string, 1024)
+
+	count := int(atomic.AddInt32(nodeCount, 1))
+
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+		startStandaloneNode(t, ctx, count, logMessages)
+	}()
+
+	log, err := waitForLogSubstring(ctx, logMessages, "started RPC server")
+	require.NoError(t, err, "RPC server didn't start")
+	var jsonLog map[string]interface{}
+	err = json.Unmarshal([]byte(log), &jsonLog)
+	require.NoError(t, err)
+
+	expectedGetStatsResponse := &rpc.GetStatsResponse{
+		Version:              "development",
+		PubSubTopic:          "/0x-orders/network/1337/version/1",
+		Rendezvous:           "/0x-mesh/network/1337/version/1",
+		PeerID:               jsonLog["myPeerID"].(string),
+		EthereumChainID:      1337,
+		LatestBlock:          rpc.LatestBlock{},
+		NumOrders:            0,
+		NumPeers:             0,
+		MaxExpirationTime:    "115792089237316195423570985008687907853269984665640564039457584007913129639935",
```

Comment:
```suggestion
		MaxExpirationTime:  constants.UnlimitedExpirationTime.String(),
```

nit: This is a little easier to understand IMO.